src/xrpld/overlay/detail/PeerImp.cpp

```diff
@@ -249,7 +249,13 @@ PeerImp::send(std::shared_ptr<Message> const& m)
 
     auto validator = m->getValidatorKey();
     if (validator && !squelch_.expireSquelch(*validator))
+    {
+        overlay_.reportTraffic(
+            TrafficCount::category::squelch_saved,
```

Comment:
@bthomee I found a way to measure the total traffic saved by squelching. We don't get the granularity, but it's better than nothing.

Reply:
> What about naming the category squelch_suppressed instead?
I think that's a good name, thanks @bthomee.

`squelch_suppressed` captures the message and byte amount that were saved by squelching. `squelch` capture the number of specifically `squelch` messages sent and received. We can't re-use `squelch` category to capture 
`squelch_suppressed` metrics as semantically they're very different, and thus would leave to confusion. In addition, `squelch` in and out are both used to report incoming and outgoing messages.